wpilibc/athena/src/Timer.cpp

```diff
@@ -41,12 +41,9 @@ double GetClock() { return Timer::GetFPGATimestamp(); }
  *         on Saturday.
  */
 double GetTime() {
-  struct timespec tp;
-
-  clock_gettime(CLOCK_REALTIME, &tp);
-  double realTime = (double)tp.tv_sec + (double)((double)tp.tv_nsec * 1e-9);
-
-  return (realTime);
+  using namespace std::chrono;
```

Comment:
the comment for this methods mentions nanosecond resolution. Does this still do that? if not it's fine but just change the docs.